index.html

```diff
@@ -412,7 +412,7 @@
             The list of valid post properties is defined by the vocabularies. This allows new vocabularies to be developed outside the development of the syntax.
           </p>
           <p>
-            Most values will be strings. If a property (such as `author` for example) references another
+            Most values will be strings. If a property (such as <a>author</a> for example) references another
```

Comment:
I'm not sure how to refer to properties consistently. Some references are links to definitions like I've changed this to be, some are wrapped in double-quotes, and some are just bare with no formatting. Bare seems the most important to change but I'm not really a fan of double-quotes either? Maybe wrap 'em in `<code>`?